src/App.tsx

```diff
@@ -1,14 +1,50 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todo, setTodo] = useState<Todo | null>(null);
+  const [filterBy, setFilterBy] = useState('all');
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    getTodos()
+      .then(result => {
+        setTodos(result);
+        setIsLoaded(true);
+      });
+  }, []);
+
+  const lowerText = (str: string) => (
```

Comment:
But this function checking is includes your string query or no. So, this`s not best name for function))